### PR TITLE
Onboarding Wizard Confirmation Modal

### PR DESCRIPTION
**Pull request by amit-sawhney**
## Summary

Adds a popup modal to confirm that a user has scheduled a meeting for onboarding because the calendly has no callback to tell if it was clicked/scheduled

## Changes

- Add modal
- Rework user flow to prevent submit without confirmation

## Screenshots

image

### CODE REVIEW

**Review by musydali — CHANGES_REQUESTED** [review]
The pop-up modal has Southside written as one word and it's actually two words so it should be South Side (my typo on the Figma designs) and then instead of the button saying "Submit" it makes more sense for it to say "Complete Sign-Up" just for validation for the user so they know what that button is going to do

File: client/src/pages/wizard/Onboard8.tsx [hunk 1]
```diff
@@ -1,32 +1,81 @@
...
       >
-        Schedule meeting
-      </Button>
+        <ModalContent className="confirmation-modal-content">
+          <Button
+            onClick={() => setModalOpen(false)}
+            className="close-icon"
+            icon
+          >
+            <img src={CloseSVG} alt="X" />
```
Comment (alicesf2) [inline client/src/pages/wizard/Onboard8.tsx:45]: I suggest you use Semantic UI React's modal with the default close icon (https://react.semantic-ui.com/modules/modal/#content-close-icon) I just asked Mustafa to move the close icons to the top right cuz it's more conventional too

**Review by alicesf2 — CHANGES_REQUESTED** [review]

**Review by musydali — APPROVED** [review]

**Review by sahithimuthyala — DISMISSED** [review]

**Review by alicesf2 — CHANGES_REQUESTED** [review]
Screen Shot 2021-03-20 at 11 14 50 PM

photograpahing -> photographing

**Review by sahithimuthyala — APPROVED** [review]
💯

**Review by alicesf2 — APPROVED** [review]
🤩

**Pull request merged**